packages/api/src/command/medical/patient/consolidated-webhook.ts

```diff
@@ -58,7 +59,15 @@ export const processConsolidatedDataWebhook = async ({
 
     // create a representation of this request and store on the DB
     const payload: PayloadWithoutMeta = {
```

Comment:
update the webhook docs to reflect return of externalID too

Reply:
Since `externalId` is optional and not something i believe we want to default to im leaving it out of the responses. The same goes for the comments below. This was the reasoning for the info to explain when they should expect to receive it. 

Open to discussing this further though

Reply:
When I recently added the optional customer webhook metadata, we added documentation showing it in the response: https://docs.metriport.com/medical-api/more-info/webhooks#patient-document-data 
I think your right and it could go either way though so whatever you think is best.

Reply:
I agree we should document this on the docs. We just need to indicate its optional and only included if the patient has an `externalId`. We should be clear about where/how our cxs have access to info.

Reply:
Do you mean to include this in the example responses @leite08 ?

Reply:
The places where we have the patient's fields being defined and exemplified, like these (not a comprehensive list):

![image](https://github.com/metriport/metriport/assets/2132564/cfc39f9b-e8fd-4071-bbc4-6cae3f0b5be4)

![image](https://github.com/metriport/metriport/assets/2132564/fda43a40-b172-442d-a36b-12da12cd8d64)